tutorials/models/3_generative_model/5_dgmg.py

```diff
@@ -784,16 +784,16 @@ def _get_next(i, v_max):
 #     for g in g_list:
 #     self.graph_prop(g)
 #
-# We can modify the code to work on a batch of graphs at once by replacing
-# these lines with the following. On CPU with a Mac machine, we instantly
-# enjoy a 6~7x reduction for the graph propagation part.
+# Modify the code to work on a batch of graphs at once by replacing
+# these lines with the following. On CPU with a macOS, you instantly

```

Comment:
What about Linux?

Reply:
Should be the same. @mufeili could you provide the hardware spec here?